commit 8a114f24c485b5029806f95bbb1329d502cb8384
Author: John Doe <jdoe@email.com>
Date:   Sun Feb 11 23:46:59 2024 +0800

    [win] Fix DWM transitions for inactive windows    2023-07-12 05:31:24    738fe871783e8e90bde132cd7014a3be2ec8fb00

diff --git a/ui/views/win/hwnd_message_handler.cc b/ui/views/win/hwnd_message_handler.cc
index e04d8ed3bf682..f55e8a51b7f72 100644
--- a/ui/views/win/hwnd_message_handler.cc
+++ b/ui/views/win/hwnd_message_handler.cc
@@ -3487,7 +3487,6 @@ void HWNDMessageHandler::PerformDwmTransition() {
 
   dwm_transition_desired_ = false;
   delegate_->HandleFrameChanged();
-  SendFrameChanged();
 }
 
 void HWNDMessageHandler::UpdateDwmFrame() {
